bundle/deploy/terraform/init.go

```diff
@@ -39,6 +38,21 @@ func (m *initialize) findExecPath(ctx context.Context, b *bundle.Bundle, tf *con
 		return tf.ExecPath, nil
 	}
 
+	// Load exec path from the environment if it matches the currently used version.
+	envTFVersion := env.Get(ctx, "DATABRICKS_TF_VERSION")
```

Comment:
Why require the version to be set?

Reply:
We are setting those env vars globally for all VSCode terminals. If users start to use CLI not provider by the extension we might get problems - external CLI can rely on a different terraform version, and we don't want to force it to use the bundled terraform binary in that case.